oak_restricted_kernel_sdk/src/mock_sdk.rs

```diff
@@ -34,7 +37,7 @@ lazy_static::lazy_static! {
   };
 }
 
-#[cfg(feature = "mock_attestation")]
+#[cfg(feature = "mock_sdk")]
```

Comment:
Isn't this entire mod already feature gated?

Reply:
Removed